### PR TITLE
CIN-1669: Back to table view link works as expected

### PR DESCRIPTION
**Pull request by cinchy-yogeshjagtap**
- changed URL for back to table link

### CODE REVIEW

**Comment by cinchy-m** [comment]
1. The “back to table” menu link is entirely missing (does not show up at all in the UI), not just broken.
2. Also noticed the table link next to the record dropdown menu doesn’t link to the right place (just links to `/metaforms`. Not sure if that's a result of running in the sandbox but in production it correctly links to the table the form is based off of.

**Comment by cinchy-yogeshjagtap** [comment]
Hi @cinchy-m , I have resolved above comments and push my changes to branch . Please verify the changes on QA environment.

**Comment by cinchy-m** [comment]
@alvin-h @cinchy-yogeshjagtap all looks good to me

**Comment by cinchy-m** [comment]
@cinchy-yogeshjagtap The link now works but only when a record is loaded. This is how it should work:

1. If a record is loaded, the link should point to the table filtered to the record that was loaded in the form (this **works**)
2. If no record is loaded, the link should point to the table with no filters applied (this is **broken**)

File: src/app/core/sidenav/sidenav.component.ts [hunk 1]
```diff
@@ -56,7 +56,7 @@ export class SidenavComponent implements OnInit, OnDestroy {
 
     this.appStateService.onRecordSelected().subscribe(resp => {
       this.rowId = resp.cinchyId;
-      this.filteredTableUrl = this.tableUrl ? `${this.tableUrl}?viewId=0&fil[Cinchy%20Id].Op=Equals&fil[Cinchy%20Id].Val=${resp}` : this.filteredTableUrl;
+      this.filteredTableUrl = !this.rowId.toString().includes('null') ? `${this.tableUrl}?viewId=0&fil[Cinchy%20Id].Op=Equals&fil[Cinchy%20Id].Val=${this.rowId}` : this.tableUrl;
```
Comment (alvin-h) [inline src/app/core/sidenav/sidenav.component.ts:59]: shouldn't we check if it's null and not convert it to a string?

Comment (cinchy-m) [reply src/app/core/sidenav/sidenav.component.ts:59]: @cinchy-yogeshjagtap ☝️

Comment (cinchy-m) [reply src/app/core/sidenav/sidenav.component.ts:59]: @alvin-h FYI, this now works as expected. Whether or not it needs the change you mention here I'll leave to you two.

**Review by alvin-h — APPROVED** [review]

**Pull request merged**